### PR TITLE
remove viewStart from getDecisions

### PR DESCRIPTION
**Pull request by ninaceban**
## Description

Removing `viewStart` flag. We will retrieve decisions by scope only. We will not retrieve page-wide scope decisions for now.
## Types of changes

- [] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [ ] I have submitted a [documentation](https://github.com/AdobeDocs/alloy-docs) pull request or no changes are needed.
- [x] I have made any necessary test changes and all tests pass.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: src/components/Personalization/index.js [hunk 1]
```diff
@@ -168,14 +167,10 @@ const createPersonalization = ({ config, logger, eventManager }) => {
 
     commands: {
       getDecisions(options = {}) {
```
Comment (artur-ciocanu) [inline src/components/Personalization/index.js]: I think you can remove the cloning, if the `scopes` parameter is not modified.

File: src/components/Personalization/index.js [hunk 2]
```diff
@@ -93,12 +92,12 @@ const createCollect = eventManager => {
 const validateOptions = options => {
   const validate = objectOf(GET_DECISIONS_OPTIONS_SCHEMA);
   const result = validate(options);
-  const { viewStart, scopes } = result;
+  const { scopes } = result;
 
-  if (!viewStart && scopes.length === 0) {
+  if (scopes.length === 0) {
     throw new Error(
       "Invalid getDecisions command options parameter: " +
```
Comment (artur-ciocanu) [inline src/components/Personalization/index.js]: I would use lower case `scopes`.

File: src/components/Personalization/index.js (hunk 1 above)
Comment (jonsnyder) [reply src/components/Personalization/index.js]: Indeed, the validation effectively clones the array.

**Comment by jfkhoury** [comment]
Looks good! Let’s merge it today if possible.

File: src/components/Personalization/index.js [hunk 3]
```diff
@@ -20,7 +20,6 @@ import * as SCHEMAS from "../../constants/schemas";
 const DECISIONS_HANDLE = "personalization:decisions";
 const PAGE_WIDE_SCOPE = "page_wide_scope";
 const GET_DECISIONS_OPTIONS_SCHEMA = {
-  viewStart: boolean().default(false),
   scopes: arrayOf(string()).default([])
```
[the remark this answers is not in the document]
Comment (ninaceban) [reply src/components/Personalization/index.js]: thank you

**Pull request merged**